test/tests/bidi.json

```diff
@@ -0,0 +1,145 @@
+{
+  "scenario": "Bidi support",
+  "description": "Tests for correct parsing of messages with bidirectional marks and isolates",
+  "defaultTestProperties": {
+    "locale": "en-US"
+  },
+  "tests": [
+    {
+        "description": "simple-message    = o [simple-start pattern]",
+        "src": "  \u061C Hello world!",
+        "exp": "  \u061C Hello world!"
+    },
+    {
+        "description": "complex-message   = o *(declaration o) complex-body o",
+        "src": "\u200E .local $x = {1} {{ {$x}}}",
+        "exp": " 1"
+    },
+    {
+        "description": "complex-message   = o *(declaration o) complex-body o",
+        "src": ".local $x = {1} \u200F {{ {$x}}}",
+        "exp": " 1"
+    },
+    {
+        "description": "complex-message   = o *(declaration o) complex-body o",
+        "src": ".local $x = {1} {{ {$x}}} \u2066",
+        "exp": "1"
```

Comment:
Missing a space before the output, since the pattern is `{{ {$x}}}`

Reply:
Oops, there were actually a few more of these, because I just found a bug in my ICU4C test runner. Fixed in [f82e12a](https://github.com/unicode-org/message-format-wg/pull/902/commits/f82e12a8a9ca7bfeafcb041c0ed02a94de90c7a9)